lib/chef/provider/subversion.rb

```diff
@@ -194,6 +196,21 @@ def authentication
         result
       end
 
+      def proxy
+        %w{http https}.each { |p|
+          if @new_resource.repository.start_with?("#{p}:") &&
+              ENV["#{p}_proxy"] != nil &&
+              !fuzzy_hostname_match_any?(URI.parse(@new_resource.repository).host, ENV["no_proxy"])
+            proxy_uri = URI.parse(ENV["#{p}_proxy"])
```

Comment:
`Chef::HTTP::BasicClient#proxy_uri` does different processing for the proxy URI, it'd be good to at least check out the difference.